framework-plugins/lisk-framework-http-api-plugin/swagger.yml

```diff
@@ -19,41 +19,22 @@ info:
     # Welcome!
 
     ## Access restrictions
-    The API endpoints are by default restricted to a whitelist of IPs that can be found under `modules.http_api.access.whiteList` in the config.
-    If it is required for your API to be accessible by the public, this can be achieved by changing `modules.http_api.access.public` to `true`.
-    This will allow anyone to make requests to your Lisk Core node.
-    However some endpoints will stay private, hence this means that only a list of whitelisted IPs can successfully make API calls to that particular endpoint;
-    this includes all forging related API calls.
-    By default, only the nodes' local IP is also included in the whitelist, nevertheless it is possible to change the setting in `config` under `modules.http_api.forging.access.whiteList`.
-    For more details, see the [Lisk Core management docs](https://lisk.io/documentation/lisk-core/3.0.0/management/) or the descriptions at the respective endpoint below.
+    The API endpoints are by default restricted to a whitelist of IPs that can be found under `modules.http_api.whiteList` in the config.
+    If it is required for your API to be accessible by the public, this can be achieved by changing `modules.http_api.whiteList` to `[]`.
+    This will allow anyone to make requests to your application.
 
     ## Requests
     Chained filter parameters are logically connected with `AND`.
     `HTTP` is the supported URL schema by default.
```

Comment:
Are we sure of keeping this behavior? In the past, we had requests to allow composite filters with different compositions.

Reply:
for now, filter only exists mainly limit and offset (except state on peer), so this should be fine for now